frontend/src/routes/Home/Search/searchDefinitions.tsx

```diff
@@ -17,49 +17,21 @@ export const getSearchDefinitions = (t: TFunction) => {
   return {
```

Comment:
⬇️ Start review at the bottom. ⬇️

Reply:
@zlayne The change set is big, but notice there's no changes to the test snapshot.  The tests for this function were great identifying problems.

Reply:
This looks great!